styles/styles.css

```diff
@@ -15,20 +15,26 @@
   --background-color: white;
   --light-color: #f8f8f8;
   --dark-color: #505050;
-  --text-color: #131313;
+  --text-color: #353535;
+  --heading-color: #000;
   --link-color: #3b63fb;
   --link-hover-color: #1d3ecf;
 
   /* fonts */
-  --body-font-family: roboto, roboto-fallback, sans-serif;
-  --heading-font-family: roboto-condensed, roboto-condensed-fallback, sans-serif;
+
+  /* --body-font-family: roboto, roboto-fallback, sans-serif;
+  --heading-font-family: roboto-condensed, roboto-condensed-fallback, sans-serif; */
+  
+  --body-font-family: "Adobe Clean", roboto-fallback, sans-serif;
+  --heading-font-family: "Adobe Clean", roboto-condensed-fallback, sans-serif;
 
   /* body sizes */
   --body-font-size-m: 22px;
   --body-font-size-s: 19px;
   --body-font-size-xs: 17px;
 
   /* heading sizes */
+  --heading-font-size-hero: 90px;
```

Comment:
This reminds me of one problem I have with the boilerplate code. I'll post a learning for everyone, but I like to use REM units. https://www.w3docs.com/snippets/css/which-is-better-to-use-in-css-px-em-or-rem.html
([see my PR for Abbie yesterday for an example](https://github.com/AbbVie/xwalkDurystasavingsProg/pull/33/files#diff-37f6471b306924bbb531d040e7ab55df5f83511246a411d22947462323516afaR138-R154))